molecules/_tables.scss

```diff
@@ -51,25 +51,14 @@ table {
     border-bottom: 1px solid $table-header-border-color;
     background-color: $table-header-background-color;
     color: $table-header-color;
-    th {
-      text-align: left;
-    }
-    tr:not(:last-child) th {
-      border-bottom: 1px solid $table-header-row-border-bottom-color;
-    }
-    tr:not(:first-child) th {
-      border-top: 1px solid $table-header-row-border-top-color;
-      background-color: darken($table-header-background-color, 3%);
-    }
-    tr:last-child th {
-      border-bottom: 1px solid $table-header-border-color;
-    }
-    tr:first-child {
+    tr {
       th {
         padding: 0 $space-xl 0 $space-m;
-        line-height: $space-xl;
+        line-height: $space-xl - .125rem; // Somehow the line height doesn't match the target 2rem height
```

Comment:
Nope. But it's not the first time that the line height results in a different container height than expected. This is probably because it's just the line and not the container height and therefore it got a different calculation.